package.json

```diff
@@ -38,7 +38,7 @@
     "format": "prettier -w . --cache"
   },
   "dependencies": {
-    "@astrojs/compiler": "^0.15.2",
+    "@astrojs/compiler": "^0.19.0",
```

Comment:
Does the compiler upgrade brings any other changes?

Reply:
I think it also comes with this one https://github.com/withastro/compiler/pull/414
It's kind of hard to keep track of what is changed in the AST from the changelog, that's why I'm not sure how many changes were made but I found only those two.